{"failedTests":[{"testName":"parallel/test-async-context-frame","timestamp":1725161465141,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2022-COMPILED_BY-vs2022 #29807","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022/29807/","buildNumber":29807,"builtOn":"test-rackspace-win2022_vs2022-x64-2","nodeVersion":"23.0.0","commitHash":"68fc01d1c35d138cb0967da6e10bb0d856fefad0","callStack":[{"upstreamBuild":65365,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73764,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61771,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54591"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":108,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (4171.1723ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (4246.1611ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (4438.2938ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (4652.1564ms)\n  ✖ async-hooks\\test-async-local-storage-enable-disable.js (4476.041ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-enable-disable.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 4)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (4430.9075ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (4911.4911ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (4527.1673ms)\n  ✖ async-hooks\\test-async-local-storage-http-agent.js (4798.7508ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-http-agent.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 8)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-http.js (4455.549ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (4245.0617ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (3967.7055ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (4678.5239ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (4430.2784ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (4643.7232ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (4681.3748ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (4760.5205ms)\n  ✔ parallel\\test-async-local-storage-bind.js (4539.8268ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (4645.3806ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (4636.5124ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (4369.9571ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (4627.46ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (4240.8339ms)\n▶ AsyncContextFrame (5297.7029ms)\nℹ tests 23\nℹ suites 1\nℹ pass 21\nℹ fail 2\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 5312.9187\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-enable-disable.js (4476.041ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-enable-disable.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 4)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-http-agent.js (4798.7508ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-http-agent.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 8)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":5796.08,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}},{"testName":"parallel/test-runner-watch-mode","timestamp":1725161465141,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win10-COMPILED_BY-vs2022 #29807","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/29807/","buildNumber":29807,"builtOn":"test-azure_msft-win10_vs2019-x64-1","nodeVersion":"23.0.0","commitHash":"68fc01d1c35d138cb0967da6e10bb0d856fefad0","callStack":[{"upstreamBuild":65365,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73764,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61771,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54591"}],"tap":{"ok":false,"name":"parallel/test-runner-watch-mode","id":704,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"▶ test runner watch mode\n  ✔ should run tests repeatedly (4075.2629ms)\n  ✔ should run tests with dependency repeatedly (3821.2777ms)\n  ✔ should run tests with ESM dependency (4054.4982ms)\n  ✔ should support running tests without a file (4040.7136ms)\n  ✖ should support a watched test file rename (3606.2147ms)\n    Error: ENOENT: no such file or directory, rename 'c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687\\test.js' -> 'c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687\\test-renamed-test.js'\n        at renameSync (node:fs:1020:11)\n        at Timeout._onTimeout (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:93:40)\n        at listOnTimeout (node:internal/timers:614:17)\n        at process.processTimers (node:internal/timers:549:7) {\n      errno: -4058,\n      code: 'ENOENT',\n      syscall: 'rename',\n      path: 'c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js',\n      dest: 'c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test-renamed-test.js'\n    }\n\n  ✖ should not throw when delete a watched test file (1.2429ms)\n    Error: EPERM, Permission denied: \\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687 '\\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687'\n        at Object.rmSync (node:fs:1246:18)\n        at rmSync (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:20:8)\n        at Object.refresh (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:35:3)\n        at TestContext.refresh (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:33:10)\n        at TestHook.runInAsyncScope (node:async_hooks:211:14)\n        at TestHook.run (node:internal/test_runner/test:887:25)\n        at TestHook.run (node:internal/test_runner/test:1157:18)\n        at node:internal/test_runner/test:806:20\n        at async Suite.runHook (node:internal/test_runner/test:804:7)\n        at async Test.run (node:internal/test_runner/test:863:9) {\n      errno: 1,\n      code: 'EPERM',\n      path: '\\\\\\\\?\\\\c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687',\n      syscall: 'rm'\n    }\n\n  ✖ should run new tests when a new file is created in the watched directory (9.3403ms)\n    Error: EPERM, Permission denied: \\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687 '\\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687'\n        at Object.rmSync (node:fs:1246:18)\n        at rmSync (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:20:8)\n        at Object.refresh (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:35:3)\n        at TestContext.refresh (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:33:10)\n        at TestHook.runInAsyncScope (node:async_hooks:211:14)\n        at TestHook.run (node:internal/test_runner/test:887:25)\n        at TestHook.run (node:internal/test_runner/test:1157:18)\n        at node:internal/test_runner/test:806:20\n        at async Suite.runHook (node:internal/test_runner/test:804:7)\n        at async Test.run (node:internal/test_runner/test:863:9) {\n      errno: 1,\n      code: 'EPERM',\n      path: '\\\\\\\\?\\\\c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687',\n      syscall: 'rm'\n    }\n\n▶ test runner watch mode (19675.7158ms)\nℹ tests 7\nℹ suites 1\nℹ pass 4\nℹ fail 3\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 19989.7662\n\n✖ failing tests:\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:185:3\n✖ should support a watched test file rename (3606.2147ms)\n  Error: ENOENT: no such file or directory, rename 'c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687\\test.js' -> 'c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687\\test-renamed-test.js'\n      at renameSync (node:fs:1020:11)\n      at Timeout._onTimeout (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:93:40)\n      at listOnTimeout (node:internal/timers:614:17)\n      at process.processTimers (node:internal/timers:549:7) {\n    errno: -4058,\n    code: 'ENOENT',\n    syscall: 'rename',\n    path: 'c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js',\n    dest: 'c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test-renamed-test.js'\n  }\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:189:3\n✖ should not throw when delete a watched test file (1.2429ms)\n  Error: EPERM, Permission denied: \\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687 '\\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687'\n      at Object.rmSync (node:fs:1246:18)\n      at rmSync (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:20:8)\n      at Object.refresh (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:35:3)\n      at TestContext.refresh (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:33:10)\n      at TestHook.runInAsyncScope (node:async_hooks:211:14)\n      at TestHook.run (node:internal/test_runner/test:887:25)\n      at TestHook.run (node:internal/test_runner/test:1157:18)\n      at node:internal/test_runner/test:806:20\n      at async Suite.runHook (node:internal/test_runner/test:804:7)\n      at async Test.run (node:internal/test_runner/test:863:9) {\n    errno: 1,\n    code: 'EPERM',\n    path: '\\\\\\\\?\\\\c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687',\n    syscall: 'rm'\n  }\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:193:3\n✖ should run new tests when a new file is created in the watched directory (9.3403ms)\n  Error: EPERM, Permission denied: \\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687 '\\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687'\n      at Object.rmSync (node:fs:1246:18)\n      at rmSync (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:20:8)\n      at Object.refresh (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:35:3)\n      at TestContext.refresh (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:33:10)\n      at TestHook.runInAsyncScope (node:async_hooks:211:14)\n      at TestHook.run (node:internal/test_runner/test:887:25)\n      at TestHook.run (node:internal/test_runner/test:1157:18)\n      at node:internal/test_runner/test:806:20\n      at async Suite.runHook (node:internal/test_runner/test:804:7)\n      at async Test.run (node:internal/test_runner/test:863:9) {\n    errno: 1,\n    code: 'EPERM',\n    path: '\\\\\\\\?\\\\c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687',\n    syscall: 'rm'\n  }"},"time":21632.134,"fullname":"parallel/test-runner-watch-mode","closingTestPoint":false}},{"testName":"parallel/test-async-context-frame","timestamp":1725161465141,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win10-COMPILED_BY-vs2022 #29807","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022/29807/","buildNumber":29807,"builtOn":"test-azure_msft-win10_vs2019-x64-2","nodeVersion":"23.0.0","commitHash":"68fc01d1c35d138cb0967da6e10bb0d856fefad0","callStack":[{"upstreamBuild":65365,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73764,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61771,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54591"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":93,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (4493.6449ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (4451.5203ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (4627.6894ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (4565.7851ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (4955.6631ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (4895.5525ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (4852.581ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (5015.4431ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (4993.4217ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (5187.213ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (5087.9846ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (5144.2075ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (5517.303ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (5162.6254ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (5099.9375ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (5108.5281ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (4700.3526ms)\n  ✔ parallel\\test-async-local-storage-bind.js (4816.306ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (4740.6628ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (4800.6356ms)\n  ✖ parallel\\test-async-local-storage-exit-does-not-leak.js (4989.9365ms)\n    AssertionError [ERR_ASSERTION]: Test parallel\\test-async-local-storage-exit-does-not-leak.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 20)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (4945.6367ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (4717.1903ms)\n▶ AsyncContextFrame (5632.5402ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 5640.1623\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ parallel\\test-async-local-storage-exit-does-not-leak.js (4989.9365ms)\n  AssertionError [ERR_ASSERTION]: Test parallel\\test-async-local-storage-exit-does-not-leak.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 20)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":5997.018,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}}]}